src/api/general-services/pipeline-manage/constructors/create-new-step.js

```diff
@@ -19,6 +19,16 @@ const createNewStep = (context, step, args) => {
     server: remoterServer,
   });
 
+  // If enabled is defined and is false.
+  if (config.enabled === false) {
+    return {
+      ...step,
+      Type: 'Pass',
+      ResultPath: null,
+
+    };
+  }
+
```

Comment:
Replace with a `Pass` state if the step is not enabled to speed up execution in those cases.

Reply:
In this case, it won't be reported as completed. Only `Tasks` are. If we change that, we will need also to filter out for the UI `EndOfMap` and `EndOfPipeline`

Reply:
hmmm, okay, I think I will just delete this change then for now.